games-action/betterspades/betterspades-0.1.6~git.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Ace of Spades client for low-end systems"
+DESCRIPTION="BetterSpades is an Ace of Spades client targeted at low end \
+systems (GL/ES 1.1). Runs on your grandmother's rig!
+
+* Replicates of the great game Ace of Spades (classic voxlap)
+* Runs on very old systems back to OpenGL 1.1 (OpenGL ES support too)
+* Shares similar if not even better performance to voxlap
+* Can run on "embedded" systems like a Steam Link
+* Open for future expansion
+* Easy to use
+"
+HOMEPAGE="https://github.com/xtreme8000/BetterSpades"
+COPYRIGHT="2016-2022 xtreme8000 and contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="5438d7bbd0f6a6cd7565c5cc2118fc4935205a3c"
+SOURCE_URI="https://github.com/win8linux/BetterSpades/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bf1ea77c57eb983149e51c961b126f50c5a89fadb1a58b76f031625c62bd1135"
+SOURCE_DIR="BetterSpades-$srcGitRev"
+ADDITIONAL_FILES="betterspades.rdef.in"
+GLOBAL_WRITABLE_FILES="
+	settings/BetterSpades/config.ini auto-merge
+	"
```

Comment:
GLOBAL_WRITABLE_FILES go between ARCHITECTURES and PROVIDES, see: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering